gulp/scripts.js

```diff
@@ -16,8 +16,10 @@ export default function scripts(done) {
 	// Get a fresh copy of the config
 	const config = getThemeConfig(true);
 
+	const useSourceMap = !isProd;
```

Comment:
Please just use ‘!isProd’ in the source map calls instead of creating a new variable.

Reply:
Done. This was my first attempt. I probably did a mistake checking the files. Thanks for the review.